src/main/java/org/cbioportal/cdd/service/internal/ClinicalDataDictionaryServiceImpl.java

```diff
@@ -144,9 +145,8 @@ private ClinicalAttributeMetadata getAlteredDefaultMetadataByColumnHeader(Map<St
     }
 
     @Override
-    public Map<String, String> forceResetCache() throws ClinicalMetadataSourceUnresponsiveException {
-        clinicalAttributesCache.resetCache(true);
-        assertCacheIsValid();
```

Comment:
forceResetCache only called from endpoint after cache is initialized. Cache validation (check for null) is no longer needed -- a failed resetCache will throw a FailedCacheRefreshException and will never reach the assertCacheIsValid.